habitat-hitl/habitat_hitl/_internal/networking/networking_process.py

```diff
@@ -25,38 +26,28 @@
 use_ssl = False
 
 networking_process = None
-exit_event = None
 
 
 def launch_networking_process(interprocess_record):
     # see multiprocessing_config to switch between real and dummy multiprocessing
 
     global networking_process
-    global exit_event
 
-    # multiprocessing.dummy.Process is actually a thread and requires special logic
-    # to terminate it.
-    exit_event = (
-        threading.Event() if multiprocessing_config.use_dummy else None
-    )
+    # we don't support dummy multiprocessing at this time
+    assert not multiprocessing_config.use_dummy
```

Comment:
It's not worth the complexity of maintaining the dummy-multiprocessing version of all this.

Reply:
Should we just remove it, and introduce it again if we need it?